relay-server/src/services/health_check.rs

```diff
@@ -43,6 +43,17 @@ pub enum Status {
     Unhealthy,
 }
 
+impl Status {
+    /// Internal helper to report/log failing health checks.
+    fn report(self, probe: &'static str) -> Self {
+        if matches!(self, Self::Unhealthy) {
+            relay_log::error!("Health check probe '{probe}' failed");
```

Comment:
nit: should we make this info/warning? Not sure whether the macro reports these to Sentry; if that's the case ship the `error`.

Reply:
I think `warn` would be the appropriate level, but they are not reported to Sentry iirc.